anta/tests/logging.py

```
@@ -484,3 +499,44 @@ def test(self) -> None:
                 self.result.is_failure(
                     f"Pattern: `{logging_entry.regex_match}` - Not found in last {logging_entry.last_number_messages} {logging_entry.severity_level} log entries"
                 )
+
+
+class VerifyBadSyslog(AntaTest):
```

Let's enhance the existing `VerifyLoggingEntries` test. In the `LoggingQuery` model, we can could do something like:
```
regex_match: RegexString | list[RegexString]
last_number_messages: Annotated[int, Field(ge=1, le=9999) | None = None
last_number_time_units: Annotated[int, Field(ge=1, le=9999) | None = None
time_unit: Literal["days", "hours", "minutes", "seconds"] = "days"
severity_level: LogSeverityLevel = "informational"
fail_on_match: bool = False
```
To follow SemVer, `last_number_messages` would take precedence over `last_number_time_units`. Depending on the inputs, you would render the appropriate `show logging ...` commands.

We can also add a validator to make sure one or the other is provided. We can also support a list of regex in the same query, so that the user can provide its own list of known bad keywords.

By default, the test should fail if the provided regex are not found (following current behavior). If `fail_on_match` is True, we actually fail if the regex are found.